indexd/drs/blueprint.py

```diff
@@ -84,6 +84,35 @@ def get_signed_url(object_id, access_id):
     return res, 200
 
 
+def create_drs_uri(did):
```

Comment:
I split it out into it's own function since it's used more than once, lmk any thoughts

Reply:
This is fine. Feels cleaner